Microsoft.DotNet.Interactive/Microsoft.DotNet.Interactive.csproj

```diff
@@ -32,13 +32,15 @@
       <PrivateAssets>all</PrivateAssets>
       <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
     </PackageReference>
+    <PackageReference Include="FSharp.Compiler.Private.Scripting" Version="$(FSharpCompilerPrivateScriptingVersion)" />
```

Comment:
Remove the `Version=` attribute; it's handled by `Directory.Build.targets`

Reply:
good eyes